client/db_test.go

```diff
@@ -38,55 +38,104 @@ func ExampleDB_Get() {
 	s, db := setup()
 	defer s.Stop()
 
-	fmt.Println(db.Get("aa").String())
+	result := db.Get("aa")
+	if result.Err != nil {
+		panic(result.Err)
+	}
+	fmt.Printf("aa=%s\n", result.Rows[0].ValueBytes())
```

Comment:
Ben, See the PR I just sent out that uses the new API in the banking app.
You can comment there if you like

On Tue, May 12, 2015 at 9:53 AM Ben Darnell <notifications@github.com>
wrote:

> In client/db_test.go
> <https://github.com/cockroachdb/cockroach/pull/991#discussion_r30135211>:
>
> > @@ -38,55 +38,104 @@ func ExampleDB_Get() {
> >  	s, db := setup()
> >  	defer s.Stop()
> >
> > -	fmt.Println(db.Get("aa").String())
> > +	result := db.Get("aa")
> > +	if result.Err != nil {
> > +		panic(result.Err)
> > +	}
> > +	fmt.Printf("aa=%s\n", result.Rows[0].ValueBytes())
>
> You can, but real code wouldn't. Real code would want the value of the
> "aa" record, not a string like "0: aa=nil".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/991/files#r30135211>.
>